cuke_cataloger.gemspec

```diff
@@ -30,4 +30,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'simplecov', '< 1.0'
   spec.add_development_dependency 'coveralls', '< 1.0'
   spec.add_development_dependency 'rainbow', '< 4.0.0'
+  spec.add_development_dependency 'pry', '~> 0.13.1'
```

Comment:
In general, don't force the latest version of a dependency and don't allow going to the next major version blindly. `< 1.0` should suffice as a limitation unless there is a particular feature of `pry` that we are wanting and so would cause us to add a lower bound to the version limit.

Looking into it, `pry` appears to be one of _those_ gems that doesn't like semantic versioning enough to ever leave `0.x` and the breaking changes listed in their changelog do suggest that they are using the minor version number as their major version number. So how about we use `< 0.14` here instead?